lineage/Analyze.py

```diff
@@ -15,13 +15,41 @@ def submit(self, fn, *args, **kwargs):
 
 
 def Analyze(X, num_states, **kwargs):
-    """ Runs a tHMM and outputs the tHMM object, state assignments, and likelihood. """
+    """ 
+    Runs a tHMM and outputs the tHMM object, state assignments, and likelihood 
+    after fitting experimental data from :func: `Analyze_list`. 
+    
+    :param X: List of objects (cells) in a lineage
+    :type X: list
+    :param num_states: The number of states to identify
+    :type num_states: int
+    :return: The tHMM object after fitting
+    :rtype: object
+    :return: State assignments for each cell in the lineage
+    :rtype: object
+    :return: The likelihood of state assignments for each cell 
+    :rtype: array
+    
```

Comment:
The LL is the log-likelihood and is not an array, it is a `float` and is the summation of LLs of all cells